core/dbt/include/global_project/macros/adapters/relations/materialized_view/create.sql

```diff
@@ -1,9 +1,10 @@
 {% macro get_create_materialized_view_as_sql(relation, sql) -%}
-    {{- log('Applying CREATE to: ' ~ relation) -}}
```

Comment:
why delete this log?

Reply:
It will get put back in in the general `get_create_sql` macro. I can leave it here though and then make the update with the other change.